scripts/variantstore/wdl/extract/hail_create_vat_inputs.py

```diff
@@ -124,37 +124,52 @@ def write_sites_only_vcf(ac_an_af_split, sites_only_vcf_path):
 
     ht = ac_an_af_rows.rows()
     ht = ht.filter(ht.alleles[1] != "*") # remove spanning deletions
+    ht = ht.checkpoint("gs://fc-eada2674-7c2b-42a6-8db3-0246872596dc/checkpoint")
     # create a filtered sites only VCF
-    hl.export_vcf(ht, sites_only_vcf_path)
+    hl.export_vcf(ht, sites_only_vcf_path, parallel='separate_header')
 
 def add_variant_tracking_info(mt, sites_only_vcf_path):
```

Comment:
this one doesn't seem to be called anymore

Reply:
Do you want to remove this method? (seems to be referenced in commented out debugging code though?)